extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -471,6 +471,7 @@ export class ProjectsController {
 
 		if (publishTarget === constants.PublishTargetType.docker) {
 			const publishToDockerSettings = await getPublishToDockerSettings(project);
+			void promptForSavingProfile(project, publishToDockerSettings);		// not awaiting this call, because saving profile should not stop the actual publish workflow
```

Comment:
Just checking, but the user has already filled everything out, right?  It's just asking whether the currently-in-use settings should be saved?

Reply:
The user may have filled the settings, and yes then it is asking to save the currently-in-use settings.
Just to add, the user may have exited the prompts early, and may not have any settings- in that case just the basic information would be saved in the publish profile.

Reply:
For wholesome clarity, I am not awaiting the save call here, because I don't want the actual "Publish" to wait. The call before this just gets the setting information, but the actual publishing happens after this call.

The reason why I chose this route was- if the user doesn't go through the complete setup prompts, it exits in between; and I want to provide a chance to the user to save the profile.